tests/functional/spec/features/validation/sum/value_source.spec.js

```diff
@@ -0,0 +1,156 @@
+import TotalAnswerPage from "../../../../generated_pages/validation_sum_against_value_source/total-block.page";
+import BreakdownAnswerPage from "../../../../generated_pages/validation_sum_against_value_source/breakdown-block.page";
+import TotalPlaybackPage from "../../../../generated_pages/validation_sum_against_value_source/number-total-playback.page";
+import SecondBreakdownAnswerPage from "../../../../generated_pages/validation_sum_against_value_source/second-breakdown-block.page";
+import SubmitPage from "../../../../generated_pages/validation_sum_against_total_equal/submit.page";
+
+const answerAndSubmitBreakdownQuestion = (breakdown1, breakdown2, breakdown3, breakdown4) => {
+  $(BreakdownAnswerPage.breakdown1()).setValue(breakdown1);
+  $(BreakdownAnswerPage.breakdown2()).setValue(breakdown2);
+  $(BreakdownAnswerPage.breakdown3()).setValue(breakdown3);
+  $(BreakdownAnswerPage.breakdown4()).setValue(breakdown4);
+  $(BreakdownAnswerPage.submit()).click();
+};
+
+const answerAndSubmitSecondBreakdownQuestion = (breakdown1, breakdown2, breakdown3, breakdown4) => {
+  $(SecondBreakdownAnswerPage.secondBreakdown1()).setValue(breakdown1);
+  $(SecondBreakdownAnswerPage.secondBreakdown2()).setValue(breakdown2);
+  $(SecondBreakdownAnswerPage.secondBreakdown3()).setValue(breakdown3);
+  $(SecondBreakdownAnswerPage.secondBreakdown4()).setValue(breakdown4);
+  $(SecondBreakdownAnswerPage.submit()).click();
+};
+
+describe("Feature: Sum of grouped answers equal to validation against answer value source ", () => {
+  beforeEach(() => {
+    browser.openQuestionnaire("test_validation_sum_against_value_source.json");
+  });
+
+  describe("Given I start a grouped answer validation survey and enter 12 into the total", () => {
+    it("When I continue and enter 3 in each breakdown field, Then I should be able to get to the total playback page", () => {
+      $(TotalAnswerPage.total()).setValue("12");
+      $(TotalAnswerPage.submit()).click();
+
+      answerAndSubmitBreakdownQuestion("3", "3", "3", "3");
+
+      expect(browser.getUrl()).to.contain(TotalPlaybackPage.pageName);
+    });
+  });
+
+  describe("Given I start a grouped answer validation survey and enter 12 into the total", () => {
+    it("When I continue to second breakdown and enter values equal to calculated summary total, Then I should be able to get to the summary page", () => {
```

Comment:
This needs work, to me it is not very clear. It is saying `enter 12 into the total` then `enter values equal to calculated summary total` but the first case of entering 12 is totally redundant. I would mention Given you have a calculated summary value of 12, When ... etc.